src/icesat2_tracks/analysis_db/B04_angle.py

```diff
@@ -1,39 +1,30 @@
-import os, sys
-
 """
-This file open a ICEsat2 track applied filters and corections and returns smoothed photon heights on a regular grid in an .nc file.
+This file open a ICEsat2 track applied filters and corrections and returns smoothed photon heights on a regular grid in an .nc file.
 This is python 3
 """
+
+import sys
+import time
+
+import numpy as np
+import pandas as pd
+import h5py
+import xarray as xr
+import matplotlib.pyplot as plt
+from matplotlib.gridspec import GridSpec
+from scipy.constants import g as G
```

Comment:
> In some files `g` is used as loop variable so `G` is used instead

Given that this is a gravitational thing, we shouldn't use $G$ in place of $g$ – Big-$G$ has a totally different meaning in this context (the [gravitational constant](https://en.wikipedia.org/wiki/Gravitational_constant), rather than [gravitational acceleration](https://en.wikipedia.org/wiki/Gravitational_acceleration), $g$) and it will be confusing to subject-matter-experts reading the code. 

Some suggestions: 
1) import the constants and use them as a namespace:
  ```suggestion
  from scipy import constants
  ```

2) If a single variable name is important, use a different name, like `from scipy.constants import g as g_accel` .

3) Or rename the existing `g` in the code to `i`, `j`, `k` or some other index variable name.

Reply:
I don't think I am in a position to make this call. @mochell any thoughts?

Reply:
I vote for option 1. 

Latter when we need to refer a the constant, we write:
`constants.g`

Reply:
We'll update the g loop variables to `g_`.